splinterd/api/static/openapi.yml

```diff
@@ -1505,53 +1505,39 @@ components:
                       type: array
                       items:
                         type: integer
+
     Circuit:
       type: object
       properties:
-        circuit_id:
```

Comment:
I'll be putting up a PR against the splinter docs repo as well to reflect these changes